libcodechecker/store.py

```diff
@@ -0,0 +1,436 @@
+                        required=False,
+                        help="Path of the suppress file to use. Records in "
+                             "the suppress file are used to mark certain "
+                             "stored analysis results as 'suppressed'. "
+                             "(Reports to an analysis result can also be "
+                             "suppressed in the source code -- please "
+                             "consult the manual on how to do so.) NOTE: The "
+                             "suppress file relies on the \"bug identifier\" "
+                             "generated by the analyzers which is "
+                             "experimental, take care when relying on it.")
+
+    parser.add_argument('-f', '--force',
+                        dest="force",
+                        default=False,
+                        action='store_true',
+                        required=False,
+                        help="Delete analysis results stored in the database "
+                             "for the current analysis run's name and store "
+                             "only the results reported in the 'input' files. "
+                             "(By default, CodeChecker would keep reports "
+                             "that were coming from files not affected by the "
+                             "analysis, and only incrementally update defect "
+                             "reports for source files that were analysed.)")
+
+    dbmodes = parser.add_argument_group("database arguments")
+
+    dbmodes = dbmodes.add_mutually_exclusive_group(required=False)
+
+    dbmodes.add_argument('--sqlite',
+                         type=str,
+                         dest="sqlite",
+                         metavar='SQLITE_FILE',
+                         default=os.path.join(
+                             util.get_default_workspace(),
+                             "codechecker.sqlite"),
+                         required=False,
+                         help="Path of the SQLite database file to use.")
+
+    dbmodes.add_argument('--postgresql',
+                         dest="postgresql",
+                         action='store_true',
+                         required=False,
+                         default=argparse.SUPPRESS,
+                         help="Specifies that a PostgreSQL database is to be "
+                              "used instead of SQLite. See the \"PostgreSQL "
+                              "arguments\" section on how to configure the "
+                              "database connection.")
+
+    pgsql = parser.add_argument_group("PostgreSQL arguments",
+                                      "Values of these arguments are ignored, "
+                                      "unless '--postgresql' is specified!")
+
+    # WARNING: '--dbaddress' default value influences workspace creation
+    # in SQLite.
+    pgsql.add_argument('--db-address',
```

Comment:
I was thinking about the `--db-host` argument name for the new `store`, instead of `--db-address`. What do you think?